python/ml4ir/base/features/feature_fns/categorical.py

```diff
@@ -317,11 +423,20 @@ def categorical_indices_from_vocabulary_file(feature_info, feature_tensor, file_
         vocabulary_keys: values of the vocabulary stated in the vocabulary_file.
     """
     vocabulary_keys, vocabulary_ids = get_vocabulary_info(feature_info, file_io)
-    num_oov_buckets = feature_info.get("feature_layer_info")["args"].get("num_oov_buckets", 1)
+    args = feature_info.get("feature_layer_info")["args"]
+
+    if "dropout_rate" in args:
```

Comment:
Do we have test that goes through both path?

Reply:
We have unit tests for `categorical_embedding_from_vocabulary_file` and `categorical_embedding_from_vocabulary_file_and_dropout`. I can add something for this.